standalone-metastore/metastore-rest-catalog/pom.xml

```diff
@@ -0,0 +1,276 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>hive-standalone-metastore</artifactId>
+    <groupId>org.apache.hive</groupId>
+    <version>4.1.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>hive-metastore-rest-catalog</artifactId>
+  <name>Hive Metastore REST Catalog</name>
+  <properties>
+    <standalone.metastore.path.to.root>..</standalone.metastore.path.to.root>
+    <maven.compiler.source>8</maven.compiler.source>
+    <maven.compiler.target>8</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <log4j2.debug>false</log4j2.debug>
+    <hive.version>${project.parent.version}</hive.version>
+    <iceberg.version>1.6.1</iceberg.version>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-standalone-metastore-server</artifactId>
+      <version>${hive.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-standalone-metastore-common</artifactId>
+      <version>${hive.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-iceberg-catalog</artifactId>
```

Comment:
`standalone-metastore` shouldn't reference any hive modules. Could we please use only native iceberg dependencies? 

Impala uses `iceberg-hive-runtime`, see https://github.com/apache/impala/blob/cfeb57c128c7f514f3433a0399966f46a49a1a4a/fe/pom.xml#L433

Reply:
I'm having difficulties running tests using the iceberg-hive-runtime; it seems that version of the class expects Confvars.METASTOREURIS to exist but it does not in our current version of HiveConf. (trying to instantiate fails with: java.lang.reflect.InvocationTargetException: null... Caused by: java.lang.NoSuchFieldError: METASTOREURIS).
What am I missing ?

Reply:
It's caused by HIVE-27925: HiveConf: unify ConfVars enum and use underscore for better readability, that was a really bad move causing plenty of compatibility issues.
ATM I don't have other ideas other than to fallback to hive's `hive-iceberg-catalog`.

Reply:
maybe once https://github.com/apache/iceberg/pull/11750 is merged we could switch back to `iceberg-hive-runtime`

Reply:
Ok, for the time being, keeping hive-iceberg-catalog.